docs/devenv_cloud.md

```diff
@@ -0,0 +1,159 @@
+# MicroShift Development Environment in Cloud
+
+## Introduction
+
+The MicroShift development environment should be set up on a **local workstation** as a collection of virtual machines.
+* The [development virtual machine](./devenv_rhel8.md#create-development-virtual-machine) used for compiling and running the MicroShift binaries and building [RHEL for Edge Installer ISO](./rhel4edge_iso.md#build-rhel-for-edge-installer-iso) images
+* The [RHEL for Edge virtual machine](./rhel4edge_iso.md#install-microshift-for-edge) used for running MicroShift in a clean, production-like environment
+* The [Ansible control node virtual machine](../ansible/README.md) used for running performance benchmark automation scripts
+
+However, depending on a local workstation hardware configuration and operating system, it may not always be possible to set up the desired development environment, as in the following cases.
+* Different hardware platform setup (i.e. `x86_64` local hardware vs `aarch64` development environment or vise versa)
+* Lack of hardware resources (i.e. not enough RAM or local storage)
+
+In such cases, it is recommended to use a public cloud for setting up the MicroShift Development environment, as described in the remainder of this document.
+
+## Public Cloud Configuration
+
+The goal is to use public cloud resources for setting up the following configuration.
+* The development machine running on a cloud virtual instance
+* The RHEL for Edge virtual machine running as a **nested** virtual instance on the development host
+* The Ansible control node virtual machine running as a **nested**  virtual instance on the development host
+
+This configuration provides for the required functionality and flexibility neccessary for the MicroShift development and testing environments.
+> The MicroShift development machine requires at least 8 cores, 16 GB of RAM and 100 GB of disk space.
+
+### AWS Cloud
+
+Start by selecting the right region providing the necessary **metal** instance types for effective support of nested virtualization. Price is the main guiding factor in the choice of a metal instance type because of the modest hardware requirements of the MicroShift development environment.
+
+For development and RHEL for Edge host setup, it is recommended to use the following instances **with** hardware virtualization support:
```

Comment:
"For a MicroShift development and testing in a RHEL for Edge host setup..."

(We don't want to suggest this is for more than dev&test.)